lib/utils.js

```diff
@@ -56,9 +43,9 @@ utils.isPath = function isPath(pattern, opts) {
  * @return {Function}
  */
 
-utils.hasPath = function hasPath(pattern, opts) {
+utils.pathContains = function(pattern) {
   return function(fp) {
-    return utils.unixify(pattern, opts).indexOf(fp) !== -1;
```

Comment:
In my local `micromatch` clone I have an unfinished `test/windows.js` that I started long ago. I really have no idea what state it's in now, but if we can add some simple confirmations based on the issues we've solved in the past that will guard against a regression here (and fail on older versions while passing now) I think it'll put both our minds at ease.